lychee-bin/src/verbosity.rs

```diff
@@ -1,202 +1,165 @@
-//! Control `log` level with a `--verbose` flag for your CLI
-//!
-//! The original source is from
-//! [this crate](https://github.com/clap-rs/clap-verbosity-flag).
-//! Modifications were made to add support for serializing the `Verbosity`
-//! struct and to add a convenience method to get the verbosity status.
+//! Easily add a `--verbose` flag to CLIs using Structopt
 //!
 //! # Examples
 //!
-//! To get `--quiet` and `--verbose` flags through your entire program, just `flatten`
-//! [`Verbosity`]:
 //! ```rust,no_run
-//! # use clap::Parser;
-//! # use clap_verbosity_flag::Verbosity;
-//! #
-//! # /// Le CLI
-//! # #[derive(Debug, Parser)]
-//! # struct Cli {
-//! #[command(flatten)]
-//! verbose: Verbosity,
-//! # }
-//! ```
+//! use clap::Parser;
+//! use clap_verbosity_flag::Verbosity;
+//!
+//! /// Le CLI
+//! #[derive(Debug, Parser)]
+//! struct Cli {
+//!     #[command(flatten)]
+//!     verbose: Verbosity,
+//! }
 //!
-//! You can then use this to configure your logger:
-//! ```rust,no_run
-//! # use clap::Parser;
-//! # use clap_verbosity_flag::Verbosity;
-//! #
-//! # /// Le CLI
-//! # #[derive(Debug, Parser)]
-//! # struct Cli {
-//! #     #[command(flatten)]
-//! #     verbose: Verbosity,
-//! # }
 //! let cli = Cli::parse();
 //! env_logger::Builder::new()
 //!     .filter_level(cli.verbose.log_level_filter())
 //!     .init();
 //! ```
 //!
-//! By default, this will only report errors.
+//! This will only report errors.
 //! - `-q` silences output
 //! - `-v` show warnings
 //! - `-vv` show info
 //! - `-vvv` show debug
 //! - `-vvvv` show trace
```

Comment:
This is how it is now, isn't it?
```suggestion
//! This will only report warnings.
//! - `-qq` silences output
//! - `-q` show errors
//! - `-v` show info
//! - `-vv` show debug
//! - `-vvv` show trace
```

Reply:
Yeah the new comments describe the current behavior. The output should not change. We only gain more flexibility for more verbose output.

Reply:
I'll commit your changes later. 👍 Thanks

Reply:
Done; thanks!